src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Character Encoding</string>
```

Comment:
Would `Data encoding` be a reasonable compromise - it implies something a "bit" technical and not something to be messed with causally - and given that Mudlet is all about responding to and handling text the word "Text" seems a bit too general here?